src/main/java/io/github/externschool/planner/repository/schedule/ScheduleEventTypeRepository.java

```diff
@@ -9,4 +9,5 @@
  */
 @Repository
```

Comment:
LOL. ``@Repository is not needed at all as the interface you declare will be backed by a proxy the Spring Data infrastructure creates and activates exception translation for anyway. So using this annotation on a Spring Data repository interface does not have any effect at all.`` https://stackoverflow.com/questions/10394857/how-to-use-transactional-with-spring-data

Reply:
it's my way that does code more readable
you can remove some redundant elements if you want it

Reply:
it's ok, i respect best practices others follow and it was funny to get to know that annotation is not necessary to use.